loops.js

```diff
@@ -16,7 +18,13 @@
 // loop...
 // loop...
 // klaar!
- // ==========================================
+// ==========================================
+let i = "loop..."
```

Comment:
Ik moest bij deze regel even goed kijken. Op zich lijkt 't me geen probleem om de tekst in een variabele te stoppen, maar ik zou dan voor een andere naam gekozen hebben. i wordt ook gebruikt voor de loop, dus dat kan verwarrend werken.